jbmc/src/java_bytecode/java_multi_path_symex_checker.cpp

```diff
@@ -13,7 +13,10 @@ goto_tracet java_multi_path_symex_checkert::build_full_trace() const
 {
   goto_tracet goto_trace = multi_path_symex_checkert::build_full_trace();
   if(options.get_bool_option("validate-trace"))
+  {
     check_trace_assumptions(goto_trace, ns);
+    log.status() << "Trace validation successful" << messaget::eom;
```

Comment:
Not sure that is that useful as it should fail with exit code 6 when it is not successful.

Reply:
It is useful for checking that the trace has been checked!